fronts-client/src/components/CollectionDisplay.tsx

```diff
@@ -133,11 +139,10 @@ const CollectionMeta = styled(CollectionMetaBase)`
 const ItemCountMeta = CollectionMetaBase;
 
 const CollectionHeadingSticky = styled.div`
-	position: sticky;
```

Comment:
It looks like this line is the cause of the change in behaviour CP emailed a few people about this morning: the header on a container used to follow the user down the container, keeping (for example) the launch button easy to access when the top of the container has scrolled offscreen, and after this change it doesn't do that any more.

Reply:
Thanks Emily, small change that was missed in review! I've put out a fix :)

Reply:
Thanks both, It’s great to see the solution in my absence on Friday, I noticed that this fix had an unintended side effect in other editions in the Fronts, 

We’ve previously considered these editions as not in use, but involvement in data curation raises an open question about whether they are still actively use via fronts tool?

I’d like to clarify this to ensure we handle them appropriately.
Looking forward to your thoughts @fredex42 @Georges-GNM